components/Common/Search/index.module.css

```diff
@@ -29,6 +29,8 @@
     absolute
     right-2
     ml-2
+    transition-opacity
+    duration-500
     md:visible
     [&>kbd]:font-ibm-plex-mono;
```

Comment:
Since we use the `kbd` element instead of the `span` element, you can make this definition directly to the element itself

Reply:
Great point! I've moved the style definition directly to the kbd element.